client/src/components/Navbar.js

```diff
@@ -54,6 +55,8 @@ const Navbar = () => {
   const dispatch = useAuthDispatch();
   const history = useHistory();
 
+  console.log("prod: ", isProd);
```

Comment:
We can remove log if not required.

Reply:
Approving the changes as only logs issue. Rest connectivity looks good. Thank you!